AdminApi/src/AdminApi/Properties/launchSettings.json

```diff
@@ -1,13 +1,14 @@
 {
-    "$schema": "http://json.schemastore.org/launchsettings.json",
-    "profiles": {
-        "Default": {
-            "commandName": "Project",
-            "environmentVariables": {
-                "ASPNETCORE_ENVIRONMENT": "Local",
-                "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.AspNetCore.SpaProxy",
-              "ASPNETCORE_URLS": "http://*:8082"
-            }
-        }
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "Default": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Local",
+        "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.AspNetCore.SpaProxy",
+        "ASPNETCORE_URLS": "http://*:8082"
+      },
+      "workingDirectory": "bin/Debug/net8.0"
```

Comment:
As I told You, the override file is copied into the target directory. In order for the app to find the file, its working directory also needs to be the target directory

Reply:
I don't quite get it. The dll is saved in `bin/Debug/net8.0`. So I would expect the working directory to already be that directory.

Reply:
It isn't. The default working directory is the source directory (at least in rider)

Reply:
Hm okay. But isn't there some kind of environment variable or similar which we can set the working directory to?
Because currently, if we update the .NET version, it will cause a lot of confusion, because it will still use the working directory of the old version. It might take a lot of time to find out the reason for this.

Reply:
Done. I used the `TargetFramework` property for the respective .net version